### PR TITLE
Resque.inline should prevent hitting Redis

### PR DESCRIPTION
**Pull request by chrisccerami**
This addresses #1544

### CODE REVIEW

File: lib/resque/data_store.rb [hunk 1]
```diff
@@ -97,7 +97,7 @@ def server_time
     end
 
     def redis_time_available?
-      @redis.time
+      @redis.time unless Resque.inline
```
Comment (hodgestar) [inline lib/resque/data_store.rb]: This returns null when inline is true, yes? Is it okay to return null instead of false? Otherwise looks good to me.

Comment (chrisccerami) [inline lib/resque/data_store.rb]: Yup, because `nil` is a falsey value in Ruby.

Comment (fw42) [inline lib/resque/data_store.rb]: Can you explain the consequences of this change? Does this mean that the server-time will never be used (even if the server supports it) unless you have inlining disabled?

Comment (chrisccerami) [inline lib/resque/data_store.rb]: Yes. From my understanding (and I may be wrong) when `Resque.inline` is set to true, Redis is bypassed entirely and all workers are run in the foreground.

**Comment by chrisccerami** [comment]
Should also address #1552

**Pull request merged**